code/WorkInProgress/MechanicMadness.dm

```diff
@@ -2714,7 +2714,7 @@
 
 	attackby(obj/item/W as obj, mob/user as mob)
 		if(..(W, user)) return 1
-		else if(istype(W, src.compatible_guns))
+		else if(istype(W, src.compatible_guns) || istype(W, /obj/item/gun/flamethrower))
```

Comment:
why not add to the list

Reply:
oh i see, it's an istype. that's super weird lol

Reply:
And now its a list!